src/iamclient/iamclient.cpp

```diff
@@ -139,6 +145,14 @@ Error IAMClient::OnNodeStatusChanged(const String& nodeID, const NodeStatus& sta
     return ErrorEnum::eNone;
 }
 
+void IAMClient::OnCertChanged(const iam::certhandler::CertInfo& info)
+{
+    LOG_DBG() << "Received on cert changed event: " << info;
```

Comment:
`Cert changed` or `Cert changed event received`.

Should be guarded with mutex.